packages/arb-token-bridge-ui/src/components/common/Tooltip.tsx

```diff
@@ -4,20 +4,24 @@ export type TooltipProps = {
   show?: boolean
   children: React.ReactNode
   content?: React.ReactNode
+  wrapperClassName?: string
+  theme?: 'light' | 'dark'
 }
 
 export function Tooltip({
   show = true,
   content,
+  wrapperClassName = 'w-max',
+  theme = 'light',
```

Comment:
Any reason why we can't have an appwide dark mode for tooltips, since it's already dark within `TransferPanelMain`?

![image](https://user-images.githubusercontent.com/7558499/198279253-ad1df781-f322-409b-a20f-a20afe86da6b.png)

Reply:
this could have been a design choice? I don't think we should question it here on the PR
let's document it to a question list on Notion for the next meeting?

Reply:
Sure, we can take it up separately. Just the icon size issue is what we need to fix now.